### PR TITLE
PS-6783 [Doc] Release Notes

### PR DESCRIPTION
**Pull request by patrickbirch**
fix to remove internal task only jira tickets from the release notes
added 5.6.47-87.0 
added 5.6.47-87.0 to index
 Changes to be committed:
	modified:   source/release-notes/Percona-Server-5.6.40-84.0.rst
	modified:   source/release-notes/Percona-Server-5.6.44-85.0.rst
	modified:   source/release-notes/Percona-Server-5.6.45-86.1.rst
	new file:   source/release-notes/Percona-Server-5.6.47-87.0.rst
	modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date| (Downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-5.6/LATEST/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+This release merges changes of `MySQL 5.6.47
+<https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-47.html>`_, including
+all the bug fixes in it. Percona Server for MySQL 5.6.47-88 is now the current
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: version is `87.0`

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- :psbug:`6750`,
+- :psbug:`4541`,
+- :psbug:`4842`,
+- :psbug:`5952`,
+- :psbug:`5956`, and
+- :psbug:`5521`
+
+
+
+Find the release notes for Percona Server for MySQL 5.6.47-88 in
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: s/88/87.0/

**Review by hrvojem — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: Fixed.

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: fixed.

**Review by kamil-holubicki — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* :psbug:`5813`: Setting 'none' value for slow_query_log_use_global_control  throws error.
+* :psbug:`4541`: Document that maximum- and minimum- option modifiers should not be used with non-numeric system variables
+* :psbug:`6750`: Red Hat Enterprise Linux 8 had a file conflict when installing client packages
+* :psbug:`5675`: wrong row is updated (Upstream :mysqlbug:`96578`)
+* :psbug:`5591`: mysql server crash after empty set sql query  (Upstream :mysqlbug:`95236`)
+* :psbug:`1673`: LP #1520467: ps_tokudb_admin cannot install tokudb remotely
+* :psbug:`5956`: Root session must not be allowed to kill Utility user session
+* :psbug:`5952`: Utility user visible in performance_schema.threads
+* :psbug:`5642`: page tracker thread does not exit if startup fails
+* :psbug:`4842`: Documentation -  innodb_corrupt_table_action does not list a default value
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst:28]: nit, extra space after '-'

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.6.47-87.0
+includes all the features and bug fixes available in
+`MySQL 5.6.47 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-47.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+
+* :psbug:`1469`: The Memory storage engine detects an incorrect  "is full"  condition stuck when the space contained reusable memory chunks and the space could be reused.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: nit, extra spaces around "is full"

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* :psbug:`6023`: Percona server exits after a Kerberos password change after the password has expired.
+* :psbug:`5813`: Setting 'none' value for slow_query_log_use_global_control  throws error.
+* :psbug:`4541`: Document that maximum- and minimum- option modifiers should not be used with non-numeric system variables
+* :psbug:`6750`: Red Hat Enterprise Linux 8 had a file conflict when installing client packages
+* :psbug:`5675`: wrong row is updated (Upstream :mysqlbug:`96578`)
+* :psbug:`5591`: mysql server crash after empty set sql query  (Upstream :mysqlbug:`95236`)
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: nit, extra space after "sql query"

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+includes all the features and bug fixes available in
+`MySQL 5.6.47 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-47.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+
+* :psbug:`1469`: The Memory storage engine detects an incorrect  "is full"  condition stuck when the space contained reusable memory chunks and the space could be reused.
+* :psbug:`6023`: Percona server exits after a Kerberos password change after the password has expired.
+* :psbug:`5813`: Setting 'none' value for slow_query_log_use_global_control  throws error.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: nit, extra space after "slow_query_log_use_global_control"

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.47-87.0.rst]: Fixed.

File: doc/source/release-notes/Percona-Server-5.6.47-87.0.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.47-87.0.rst:28]: fixed

**Pull request merged**